components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -653,6 +653,19 @@ def leave_none_unset_int(s):
           " ] or [\"Repository\", {\"viewname\": \"webindex\", "
           "\"query_string\": {\"experimenter\": -1}}, "
           "{\"title\": \"Repo\"}]'``")],
+    "omero.web.ui.metadata_pane":
```

Comment:
this allows to set custom order of metadata pane accordeon. Any suggestions about var naming, dict, etc. see https://github.com/openmicroscopy/openmicroscopy/pull/4704/commits/ea068154e8a62966af4ee4e388730d5b8f62fdf8#diff-5ecd969a9790526e02d15fcb673d6ca5R722 for how template use it